src/packer.pkr.hcl

```diff
@@ -74,9 +75,65 @@ data "amazon-ami" "debian_bookworm" {
   region      = var.build_region
 }
 
+data "amazon-ami" "debian_bookworm_arm64" {
```

Comment:
Would you please put this `data` resource before the x86_64 one to maintain the same order as the `source` resources below?

Reply:
Please see commit [6642ba3](https://github.com/cisagov/skeleton-packer/pull/355/commits/6642ba3ed3be6f945adb80f6da611c1986f421d6).